docs/Deployment/centralizedConfiguration.md

```diff
@@ -65,47 +65,67 @@ The Consul configurations are stored in a local volume. When cleaning up contain
     CONSUL_PORT=8500
     ```
 
-3. Pull the images.
+3. Recreate the MPS and RPS containers to update their configuration.
 
     === "Linux"
         ```
-        sudo docker-compose pull
+        sudo docker compose up -d
         ```
     
     === "Windows"
         ```
-        docker-compose pull
+        docker compose up -d
         ```
 
-4.  Start the containers.
+4. Pull the Consul image.
+
+    === "Linux"
```

Comment:
something to think about --- the difference of sudo in these commands is hardly enough in my mind to justify a tab, can simply be  the same as windows.

Reply:
I've been debating for a while.  Maybe it's finally time to test it.  I'll cut it from the docs and in the Getting Started Guide have a blurb about sudo and point to Docker docs how to not have to type it for more newbie Docker Linux users.